addons/calendar/data/mail_template_data.xml

```diff
@@ -44,19 +44,6 @@
         </t>
 
     </p>
-    <div style="text-align: center; padding: 16px 0px 16px 0px;">
```

Comment:
These buttons are present in multiple templates targeting the "calendar.attendee" model, they probably need to all be cleaned as "_notify_get_recipients_groups" will affect all of them.

Meaning the Date: Updated template probably has the buttons twice?